lib/routes.js

```diff
@@ -0,0 +1,26 @@
+var express  = require('express');
+var router   = express.Router();
```

Comment:
you're implementation is great and will work. so +1 on this.

as an alternative implementation think about what would happen if we were to have multiple route files defining different routes - different resources ? - not that I am necessarily in favour of this approach specially not in small apps however I have seen use cases for them where this would be applicable. We would be requiring express in each file - we already have required in the server.js file so why not reuse existing.

As another note ... in the scenario above we would have to do another app.use(anotherRoutesFile) meaning another routes file required with again the router instantiated again within it. We want to ideally use one router throughout.

Again like I mentioned ... this approach is great and works well and cleaner than some of the other attempts by others, but the aim is to make the code as modular and generic as possible with the least amount of repetition

great stuff 👍